pkg/log/logger.go

```diff
@@ -305,3 +305,13 @@ func NewLogger(opts ...Option) (*Logger, error) {
 	logger.core = zapcore.NewCore(enc, writer, level)
 	return logger, nil
 }
+
+func getTimeEncoder(o *options, defaultEncoder func(t time.Time, enc zapcore.PrimitiveArrayEncoder)) func(t time.Time, enc zapcore.PrimitiveArrayEncoder) {
```

Comment:
We don't need to pass the `defaultEncoder`. In fact, let `getTimeEncoder` decides the default time encoder.


```go
func getTimeEncoder(o *options) func(t time.Time, enc zapcore.PrimitiveArrayEncoder) {
}
```

Reply:
Get it.